crates/node_binding/src/plugins/mod.rs

```diff
@@ -100,6 +108,27 @@ impl rspack_core::Plugin for JsHooksAdapter {
       .map_err(|err| internal_error!("Failed to call make: {err}",))?
   }
 
+  async fn normal_module_factory_resolve_for_scheme(
+    &self,
+    _ctx: rspack_core::PluginContext,
+    args: &NormalModuleFactoryResolveForSchemeArgs,
+  ) -> PluginNormalModuleFactoryResolveForSchemeOutput {
+    let res = self
```

Comment:
Could you add this to the hook disabled map? I think it will cause a severe performance regression.

Reply:
I miss that part, will do.

Reply:
Tracked here.

Reply:
https://github.com/web-infra-dev/rspack/issues/2400